src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectProtocolValidator.cs

```diff
@@ -433,24 +433,27 @@ public virtual HashAlgorithm GetHashAlgorithm(string algorithm)
                     case SecurityAlgorithms.ECDSA_SHA256:
                     case SecurityAlgorithms.HMAC_SHA256:
                     case SecurityAlgorithms.RSA_SHA256:
+                    case SecurityAlgorithms.RSASSA_PSS_SHA256:
                     case SecurityAlgorithms.RsaSha256Signature:
-                    case SecurityAlgorithms.PS256:
+                    case SecurityAlgorithms.EcdsaSha256Signature:
```

Comment:
It's **really** time to fix these inconsistent names (XMLDSig URN, JWA short names...), really :smile: